tests/Common/DevBranchesTest.php

```diff
@@ -14,32 +14,60 @@ class DevBranchesTest extends StorageApiTestCase
     private static $cleanupAfterClassTokenId;
 
     private static $teardownClient;
+
     /**
      * @dataProvider provideValidClients
      */
     public function testCreateBranch(Client $providedClient)
     {
-        $providedClient->verifyToken();
+        $providedToken = $providedClient->verifyToken();
         $branches = new DevBranches($providedClient);
 
-        $branchName = __CLASS__ . ' příliš žluťoučký kůň' . microtime();
-        $branch = $branches->createBranch($branchName);
+        // cleanup
+        $branchesList = $branches->listBranches();
+        $branchName = __CLASS__ . '\\' . $this->getName() . '\\' . $providedToken['id'];
+        $branchesCreatedByThisTestMethod = array_filter(
+            $branchesList,
+            function ($branch) use ($branchName) {
+                return strpos($branch['name'], $branchName) === 0;
+            }
+        );
+        foreach ($branchesCreatedByThisTestMethod as $branch) {
+            $branches->deleteBranch($branch['id']);
+        }
 
-        $this->assertArrayHasKey('created', $branch);
-        unset($branch['created']);
+        // test
+
+        // can create branch
+        $branch = $branches->createBranch($branchName);
         $this->assertArrayHasKey('id', $branch);
-        $branchId = $branch['id'];
-        unset($branch['id']);
+        $this->assertArrayHasKey('name', $branch);
+        $this->assertArrayHasKey('created', $branch);
+        $this->assertArrayNotHasKey('admin', $branch);
```

Comment:
Preco sa tu testuje ze tam nieje `admin` nevidim nikde v PR na connection ze by to tam predtim bolo a ten PR to odobera

Reply:
Protože je to v DB - ale v rámci KBC-866 a KBC-867 se to bude odebírat.